content/articles/understanding-cross-site-scripting-attacks/index.md

```diff
@@ -1,120 +1,111 @@
-Cross-Site Scripting (XSS) attacks are a type of injection, in which malicious scripts are injected into otherwise benign and trusted websites. XSS attacks occur when an attacker uses a web application to send malicious code, generally in the form of a browser side script, to a different end-user. Flaws that allow these attacks to succeed are quite widespread and occur anywhere a web application uses input from a user within the output it generates without validating or encoding it.
+Cross-Site Scripting (XSS) attacks are a type of injection, in which malicious scripts are injected into otherwise benign and trusted websites. XSS attacks occur when an attacker uses a web application to send malicious code, generally in the form of a browser side script, to a different end-user. Flaws that allow these attacks to succeed are quite widespread and occur anywhere a web application uses input from a user. This occurs when the application generates output without validating or encoding it.
 
 *(Source: [OWASP](https://owasp.org/www-community/attacks/xss/#))*
 
-According to the OWASP Top 10 of 2017 XSS is the second most prevalent issue in the OWASP Top 10, and is found in around two-thirds of all applications.
+The Open Web Application Security Project publishes the OWASP Top 10 report. It is a list of the most serious web application security concerns. In the OWASP Top 10 of 2017, XSS is the second most prevalent issue and is found in around two-thirds of all applications.
 
 *(Source: [OWASP](https://owasp.org/www-project-top-ten/2017/A7_2017-Cross-Site_Scripting_(XSS)))*
 
-The Open Web Application Security Project publishes the OWASP Top 10 report. It is a list of the most serious web application security concerns.
-
 In this article, we'll go through XSS attacks in detail, their impact, and how to mitigate them.
 
-> **Disclaimer:** - This article is for educational purposes only.
+> **Disclaimer:** This article is for educational purposes only.
 
 ### Prerequisites
+
 - Basic knowledge of HTML and Javascript
 - Basic knowledge of 
```

Comment:
I forgot to comment on this part: please finish this sentence it seems you forgot something here

Reply:
hi @john-amiscaray 

i made a mistake of git pulling from master. I think i might have lost my work.
i cant even get into the branch.  Any idea what I can do

Reply:
I have the article copied out tho.

Reply:
Fixed. Kindly go through